content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,187 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+This work follows the derivation in :cite:`ward1988` and is supported by interactive apps developed in a `binder`_.
+
+.. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+
+
+
+
+Introduction
+------------
+
```

Comment:
Minor thing, but I would like to see the intro sentence here. ie. 'The problem setup is shown in Figure ...' and then after the figure you can have the bullet points explaining in more detail what we will be doing. It is helpful to have something to give the reader a bit of context